### ISSUE TITLE
NotSupportedError: CustomElementRegistry.define: 'my-relative-time' has already been defined as a custom element

### ISSUE THREAD

**Issue by ericvb** [nielsfaber/scheduler-card#505]
### Checklist

- [X] I checked for similar existing issues (both open and closed) before posting.
- [X] I will participate in further discussion about this issue and can help by testing (if requested).

### Card Version

v2.3.0

### Component Version

v3.2.6

### Bug description

```
Logger: frontend.js.latest.202112291
Source: components/system_log/__init__.py:189
First occurred: 17:37:42 (1 occurrences)
Last logged: 17:37:42

http://x.y.z:8123/hacsfiles/scheduler-card/scheduler-card.js:43:0 NotSupportedError: CustomElementRegistry.define: 'my-relative-time' has already been defined as a custom element
```

### Steps to reproduce

I did just install the component and scheduler card.
I've not yet configured or used it.

### Additional info
Running in Home Assistant OS version 2021.12.9

**Comment by nielsfaber** [comment]
For me (and many others) this doesn't occur, so the mentioned reproduction steps are not enough for me to help you.
Did you try erasing your browser cache?
It seems that your browser seems to detect multiple instances of scheduler-card, which causes this conflict.
Otherwise there might be some other custom card creating such issue, do you have any?

**Comment by ericvb** [comment]
I deleted the browser cache (Firefox): same error
Tested with Chrome and Edge : same error
On the DevTools:
![Home Assistant OS 2012 12 10](https://user-images.githubusercontent.com/3419875/150190550-c2c6efc3-fbb9-4fe7-9c43-f5e813061c5f.png)

Searched on the name 'my-relative-time' to see if another card should use the same name : found nothing, only the scheduler-card.js file

I deinstalled the front-end of scheduler-card and restarted HA. (scheduler-component was not deinstalled)
No error in the log file
When typing customElements.get('my-relative-time') in the browser console : undefined
So no other card is using this element.

I reinstalled the front-end and restarted everything.
No error anymore....
![Home Assistant OS 2012 12 10 ok](https://user-images.githubusercontent.com/3419875/150194602-e8688fea-a417-4e81-902b-758a3f175dc4.png)

Don't know what caused this, but the error is gone...
